travis-script.sh

```diff
@@ -149,6 +149,7 @@ fi
 
 timed cabal new-build $jobs $CABAL_INSTALL_FLAGS \
                       cabal-install:cabal \
+                      cabal-install:integration-tests \
```

Comment:
They were actually removed in 2.0 as well (9368717f919326b0dcd3e03bbe43d4aa99624ef8).